targets/mega65/uart_monitor.c

```diff
@@ -268,21 +323,15 @@ static void execute_command ( char *cmd )
 			cmd = parse_hex_arg(cmd, &par1, 0, 0xFFFFFFF);
 			bank = par1 >> 16;
 			if (cmd && check_end_of_command(cmd, 1)) {
-				if (bank == 0x777)
-					m65mon_dumpmem16(par1);
-				else
-					m65mon_dumpmem28(par1);
+				m65mon_dumpmem28(par1);
 			}
 			break;
 		case 'M':
 			cmd = parse_hex_arg(cmd, &par1, 0, 0xFFFFFFF);
 			bank = par1 >> 16;
 			if (cmd && check_end_of_command(cmd, 1)) {
-				for (int k = 0; k < 32; k++) {
-					if (bank == 0x777)
-						m65mon_dumpmem16(par1);
-					else
-						m65mon_dumpmem28(par1);
+				for (int k = 0; k < 16; k++) {
```

Comment:
While it's true that the old 'M' command would spit out 32 lines, the latest 'M' command on hardware only spits out 16 lines.